lib/defaultSongObj.js

```diff
@@ -0,0 +1,18 @@
+var Language = require('../models/language')
+
+var song = {
+    title: '',
+    author: '',
+    translator: '',
+    year: '',
+    copyright: 'private',
+    youtubeLink: '',
+    lyric: []
+}
+
+Language.findOne({
```

Comment:
I'm concerned about whether or not this will work...
http://stackoverflow.com/questions/20238829/asynchronous-nodejs-module-exports